packages/nimble-components/src/listbox/tests/listbox-matrix.stories.ts

```diff
@@ -0,0 +1,32 @@
+import { ViewTemplate, html } from '@microsoft/fast-element';
+import type { Meta, StoryFn } from '@storybook/html';
+import {
+    createMatrix,
+    sharedMatrixParameters
+} from '../../utilities/tests/matrix';
+import { createMatrixThemeStory } from '../../utilities/tests/storybook';
+import { listboxTag } from '..';
+import { listOptionTag } from '../../list-option';
+
+const metadata: Meta = {
+    title: 'Tests/Listbox',
+    parameters: {
+        ...sharedMatrixParameters()
+    }
+};
+
+export default metadata;
+
+const component = (): ViewTemplate => html`
+    <${listboxTag}>
```

Comment:
We should set the selected item to be one of the options so that the test exercises the styling of a selected item.